.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: node_js
+sudo: false
+branches:
+  only:
+  - gh-pages
+env:
+  global:
+  - URL="https://rawgit.com/w3c/resource-timing/gh-pages/W3CTRMANIFEST"
```

Comment:
The URL should be changed to "https://w3c.github.io/resource-timing/W3CTRMANIFEST" after the merge :)